deploy/base/node/node_setup.yaml

```diff
@@ -35,7 +35,7 @@ metadata:
 rules:
   - apiGroups: [""]
     resources: ["pods"]
-    verbs: ["get"]
+    verbs: ["get", "watch", "list"]
```

Comment:
please make an internal bug so that we track adding this rbac as part of the release where the change goes in

Reply:
Done.